app/views/users/mailer/unlock_instructions.html.erb

```diff
@@ -1,7 +1,7 @@
-<p>Hello <%= @resource.email %>!</p>
+<p><%= t('.greeting', :recipient => @resource.email, :default => "Hello #{@resource.email}!") %></p>
 
-<p>Your account has been locked due to an excessive number of unsuccessful sign in attempts.</p>
+<p><%= t('.message', :default => "Your account has been locked due to an excessive amount of unsuccessful sign in attempts.") %></p>
 
-<p>Click the link below to unlock your account:</p>
+<p><%= t('.instruction', :default => "Click the link below to unlock your account:") %></p>
 
-<p><%= link_to 'Unlock my account', unlock_url(@resource, unlock_token: @token) %></p>
+<p><%= link_to t('.action', :default => "Unlock my account"), unlock_url(@resource, :unlock_token => @resource.unlock_token) %></p>
```

Comment:
I would prefer the new style of ruby hashes (`{ default: 'test' }` instead of `{ :default => 'test' }`) in all the html files